### ISSUE TITLE
404 on some IMDB id's

### ISSUE THREAD

**Issue by attilabogar** [itsmehemant7/PyMovieDb#6]
I think this library is awesome, however I ran into problems with some IMDB id's.

I applied the imdb.get_by_id() against 575 movies I have metadata.

13% of the results were status=404, however the ID's were correct.

Here's a list to help debugging the issue:

IMBD failed on tt0118556 / Addicted.to.Love.1997.nfo
IMBD failed on tt0058586 / A.Shot.in.the.Dark.1964.nfo
IMBD failed on tt0034492 / Bambi.1.1942.nfo
IMBD failed on tt0118689 / Bean.1997.nfo
IMBD failed on tt1529569 / Beautiful.Lies.2010.nfo
IMBD failed on tt0054698 / Breakfast.at.Tiffany's.1961.nfo
IMBD failed on tt5716464 / Breathe.2017.nfo
IMBD failed on tt0049038 / Bus.Stop.1956.nfo
IMBD failed on tt0056923 / Charade.1963.nfo
IMBD failed on tt0042332 / Cinderella.1950.nfo
IMBD failed on tt0038427 / Courage.of.Lassie.1946.nfo
IMBD failed on tt3660770 / Disney.Monkey.Kingdom.nfo
IMBD failed on tt0765128 / Disney.Oceans.nfo
IMBD failed on tt0127536 / Elizabeth.1998.nfo
IMBD failed on tt0266543 / Finding.Nemo.2003.nfo
IMBD failed on tt0083987 / Gandhi.1982.nfo
IMBD failed on tt0093209 / Hope.and.Glory.1987.nfo
IMBD failed on tt2100546 / Ice.Age-A.Mammoth.Christmas.nfo
IMBD failed on tt2718492 / Ida.2013.nfo
IMBD failed on tt2073128 / IMAX.To.the.Arctic.nfo
IMBD failed on tt0329737 / It's.a.Very.Merry.Muppet.Christmas.Movie.2002.nfo
IMBD failed on tt1619029 / Jackie.2016.nfo
IMBD failed on tt0058150 / James.Bond.007.1964.Goldfinger.nfo
IMBD failed on tt0064757 / James.Bond.007.1969.On.Her.Majesty's.Secret.Service.nfo
IMBD failed on tt0070328 / James.Bond.007.1973.Live.and.Let.Die.nfo
IMBD failed on tt0076752 / James.Bond.007.1977.The.Spy.Who.Loved.Me.nfo
IMBD failed on tt0113189 / James.Bond.007.1995.Goldeneye.nfo
IMBD failed on tt0246460 / James.Bond.007.2002.Die.Another.Day.nfo
IMBD failed on tt0830515 / James.Bond.007.2008.Quantum.of.Solace.nfo
IMBD failed on tt2379713 / James.Bond.007.2015.Spectre.nfo
IMBD failed on tt0118308 / Jane.Austen.Emma.1996.nfo
IMBD failed on tt0114388 / Jane.Austen.Sense.and.Sensibility.1995.nfo
IMBD failed on tt1229822 / Jane.Eyre.2011.nfo
IMBD failed on tt0274166 / Johnny.English.2003.nfo
IMBD failed on tt0048280 / Lady.and.the.Tramp.1955.nfo
IMBD failed on tt0351283 / Madagascar.1.2005.nfo
IMBD failed on tt1365483 / Main.Street.2010.nfo
IMBD failed on tt0405159 / Million.Dollar.Baby.2004.nfo
IMBD failed on tt4824302 / Mother's.Day.2016.nfo
IMBD failed on tt0203009 / Moulin.Rouge.2001.nfo
IMBD failed on tt1078912 / Night.at.the.Museum.2.Battle.of.the.Smithsonian.2009.nfo
IMBD failed on tt0032910 / Pinocchio.1940.nfo
IMBD failed on tt0377107 / Proof.2005.nfo
IMBD failed on tt0448694 / Puss.in.Boots.2011.nfo
IMBD failed on tt0070608 / Robin.Hood.1973.nfo
IMBD failed on tt0047437 / Sabrina.1954.nfo
IMBD failed on tt2872750 / Shaun.the.Sheep.Movie.2015.nfo
IMBD failed on tt1079448 / Snow.Buddies.2008.nfo
IMBD failed on tt0281373 / Snow.Dogs.2002.nfo
IMBD failed on tt0029583 / Snow.White.and.the.Seven.Dwarfs.1937.nfo
IMBD failed on tt1047011 / Song.for.Marion.2012.nfo
IMBD failed on tt0118715 / The.Big.Lebowski.1998.nfo
IMBD failed on tt0878804 / The.Blind.Side.2009.nfo
IMBD failed on tt0112579 / The.Bridges.of.Madison.County.1995.nfo
IMBD failed on tt0372824 / The.Chorus.2004.nfo
IMBD failed on tt0082474 / The.Great.Muppet.Caper.1981.nfo
IMBD failed on tt1454029 / The.Help.2011.nfo
IMBD failed on tt0061852 / The.Jungle.Book.1967.nfo
IMBD failed on tt2555736 / The.Second.Best.Exotic.Marigold.Hotel.2015.nfo
IMBD failed on tt2709768 / The.Secret.Life.of.Pets.1.2016.nfo
IMBD failed on tt0159097 / The.Virgin.Suicides.1999.nfo
IMBD failed on tt1441912 / The.Way.2010.nfo
IMBD failed on tt0209475 / The.Wedding.Planner.2001.nfo
IMBD failed on tt0430770 / The.Women.2008.nfo
IMBD failed on tt0117951 / Trainspotting.1996.nfo
IMBD failed on tt0358273 / Walk.the.Line.2005.nfo
IMBD failed on tt0910970 / Wall.E.2008.nfo

**Comment by attilabogar** [comment]
Looks like incoherent behaviour.
I re-run the query against the failed ID's and found only 23 failed out of the previous 72 failed ones.

**Comment by itsmehemant7** [comment]
@attilabogar did you put a delay between requests?

**Comment by attilabogar** [comment]
Yes - I did a sleep between the requests. The interesting thing, is that I re-crawled the missing ones every next day.
Now only 6 remains, these are always keep failing so might be a good edge case to debug this issue.

IMDB failed on tt0583573 / series/Friends/Friends.S08E20.nfo
IMDB failed on tt0583561 / series/Friends/Friends.S09E07.nfo
IMDB failed on tt0588158 / series/Gilmore.Girls/Gilmore.Girls.S01E07.nfo
IMDB failed on tt0888170 / series/Gilmore.Girls/Gilmore.Girls.S07E09.nfo
IMDB failed on tt0894094 / series/Gilmore.Girls/Gilmore.Girls.S07E10.nfo
IMDB failed on tt0094525 / series/Poirot/tvshow.nfo
